src/ray/raylet/worker.h

```diff
@@ -154,6 +155,12 @@ class Worker : public WorkerInterface {
   rpc::WorkerType GetWorkerType() const;
   void MarkDead();
   bool IsDead() const;
+  /// Kill the worker.
+  /// \param io_service for scheduling the graceful period timer.
+  /// \param force true to kill immediately, false to give time for the worker to
+  /// clean up and exit gracefully.
+  /// \return Void.
+  void Kill(instrumented_io_context &io_service, bool force = false);
```

Comment:
Document if `Kill` is idempotent

I think it is currently not because the PID could be reused after the process exits.

Reply:
Sounds good, I was going to suggest the same :)